templates/signup.html

```diff
@@ -203,6 +209,11 @@ <h4 class="text-blue-500 mt-0 mb-0 text-center">{{_('create_student_account_expl
                             <label for="pair_with_teacher">{{_('pair_with_teacher')}}</label>
                             <input type="checkbox" id="pair_with_teacher" name="pair_with_teacher">
                         </div>
+                        <div class="flex justify-between items-center">
+                            <label for="connect_guest_teacher">{{_('connect_guest_teacher')}}</label>
+                            <input type="checkbox" id="connect_guest_teacher" name="connect_guest_teacher"
+                            _="on click toggle .hidden on #phone_number">
```

Comment:
```suggestion
                            _="on click toggle [@required='true'] on #phone
                                     toggle .hidden on #phone_number">
```

Reply:
Ah perfect, couldn't find how to do it in htmx! Thanks a lot